docs/usage/ingress.md

```diff
@@ -193,3 +193,41 @@ kubectl delete -n argocd -f examples/argocd-ingress.yaml
 
 [install_olm]:../install/olm.md
 [docs_argo]:https://argoproj.github.io/argo-cd/getting_started/#creating-apps-via-cli
+
+### Host for Ingress in Argo CD Status
+
+When setting up access to Argo CD via an Ingress, one can easily retrieve hostnames used for accessing the Argo CD installation through the ArgoCD Operand's `status` field. To expose the `host` field, run `kubectl edit argocd argocd` and then edit the Argo CD instance server to have ingress enabled as `true`, like so: 
+
+```yaml
+server:
+    autoscale:
+      enabled: false
+    grpc:
+      ingress:
+        enabled: false
+    ingress:
```

Comment:
Setting server.grpc.ingress.enabled to`false` is not wrong. The default seems to be `false`. If it is set to `true`, the operator will create an Ingress for grpc protocol to reach the ArgoCD server which is used by CLI. The hostname of the Ingress will be shown in the `status`. The question is should the example illustrate the rpc ingress with other ingresses.